From 2c32e2ece589c504835e76d11ecc8dae30fb67e3 Mon Sep 17 00:00:00 2001
From: Roel Janssen <roel@gnu.org>
Date: Thu, 15 Feb 2018 10:25:59 +0100
Subject: [PATCH] Use escaped tabs for the read group.

---
 templates/Map.sh.tt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/Map.sh.tt b/templates/Map.sh.tt
index 1cea5be..19539d6 100644
--- a/share/hmf-pipeline/templates/Map.sh.tt
+++ b/share/hmf-pipeline/templates/Map.sh.tt
@@ -9,7 +9,7 @@ cd "[% dirs.mapping %]"
 [% opt.BWA_PATH %]/bwa mem \
     -t [% opt.MAPPING_THREADS %] \
     [% opt.MAPPING_SETTINGS %] \
-    -R "@RG	ID:[% fastq.coreName %]	SM:[% fastq.sampleName %]	PL:ILLUMINA	LB:[% fastq.sampleName %]	PU:[% fastq.flowcellID %]" \
+    -R "@RG\tID:[% fastq.coreName %]\tSM:[% fastq.sampleName %]\tPL:ILLUMINA\tLB:[% fastq.sampleName %]\tPU:[% fastq.flowcellID %]" \
     "[% opt.GENOME %]" \
     "[% fastq.R1 %]" \
     [%- IF fastq.exists('R2') %]
-- 
2.15.0

