diff -urB pipeline-4.8-orig/lib/HMF/Pipeline/Functions/Sge.pm pipeline-4.8/lib/HMF/Pipeline/Functions/Sge.pm
--- a/lib/perl5/site_perl/5.30.0/HMF/Pipeline/Functions/Sge.pm	2019-01-04 12:35:03.000000000 +0100
+++ b/lib/perl5/site_perl/5.30.0/HMF/Pipeline/Functions/Sge.pm	2020-05-28 11:31:05.546817493 +0200
@@ -44,10 +46,9 @@
     my ($opt, $function) = @_;
 
     # KODU: h_vmem setting leads to issues with Queue.jar wants to spawn jobs
-    # my $h_vmem = (4 + $opt->{$function."_MEM"})."G";
-    my $qsub = generic($opt, $function);
-    $qsub =~ s/^qsub//;
-    return $qsub;
+    my $vmem = (7 + $opt->{$function."_MEM"})."000";
+    my $slurm = qq|--cpus-per-task=$opt->{$function . "_THREADS"} --mem=$vmem --time=$opt->{$function . "_TIME"}|;
+    return $slurm;
 }
 
-1;
\ No newline at end of file
+1;
