Only in pipeline-4.8/: data
diff -crB pipeline-4.8-orig/lib/HMF/Pipeline/Functions/Validate.pm pipeline-4.8/lib/HMF/Pipeline/Functions/Validate.pm
*** a/lib/perl5/site_perl/5.28.0/HMF/Pipeline/Functions/Validate.pm	2019-01-04 12:35:03.000000000 +0100
--- b/lib/perl5/site_perl/5.28.0/HMF/Pipeline/Functions/Validate.pm	2019-04-04 12:04:47.693922926 +0200
***************
*** 442,448 ****
                  GRIDSS_BWA_BASES_PER_BATCH => \&key_not_present,
                  GRIDSS_BLACKLIST => \&missing_file,
                  GRIDSS_CONFIG => \&key_not_present,
!                 GRIDSS_PON => \&missing_directory,
  
                  TABIX_PATH => \&missing_directory,
              }
--- 442,448 ----
                  GRIDSS_BWA_BASES_PER_BATCH => \&key_not_present,
                  GRIDSS_BLACKLIST => \&missing_file,
                  GRIDSS_CONFIG => \&key_not_present,
!                 GRIDSS_PON => \&key_not_present,
  
                  TABIX_PATH => \&missing_directory,
              }
diff -crB pipeline-4.8-orig/scripts/gridss_somatic_filter.R pipeline-4.8/scripts/gridss_somatic_filter.R
*** a/share/hmf-pipeline/scripts/gridss_somatic_filter.R	2019-01-04 12:35:03.000000000 +0100
--- b/share/hmf-pipeline/scripts/gridss_somatic_filter.R	2019-04-04 12:58:38.224933170 +0200
***************
*** 18,24 ****
    print(argp)
    stop(msg)
  }
! if (is.na(argv$pondir)) {
    argv$pondir = NULL
  } else if (!dir.exists(argv$pondir)) {
    msg = paste(argv$pondir, "not found")
--- 18,24 ----
    print(argp)
    stop(msg)
  }
! if (is.na(argv$pondir) || argv$pondir == "NA") {
    argv$pondir = NULL
  } else if (!dir.exists(argv$pondir)) {
    msg = paste(argv$pondir, "not found")
***************
*** 76,83 ****
  }
  
  write(paste0("Filtering pass 1 ", argv$input), stderr())
! bpfiltered = gridss_breakpoint_filter(full_bpgr, full_vcf, pon_dir=argv$pondir, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
! befiltered = gridss_breakend_filter(full_begr, full_vcf, pon_dir=argv$pondir, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
  # shadow breakpoint removed due to initial mapq20 filter reducing FP rate
  # bpfiltered = .addFilter(bpfiltered, "shadow", is_shadow_breakpoint(bpgr, begr, full_vcf))
  
--- 76,91 ----
  }
  
  write(paste0("Filtering pass 1 ", argv$input), stderr())
! 
! if (is.null(argv$pondir)) {
!   write("Skipping PON filter", stderr())
!   bpfiltered = gridss_breakpoint_filter(full_bpgr, full_vcf, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
!   befiltered = gridss_breakend_filter(full_begr, full_vcf, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
! } else {
!   bpfiltered = gridss_breakpoint_filter(full_bpgr, full_vcf, pon_dir=argv$pondir, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
!   befiltered = gridss_breakend_filter(full_begr, full_vcf, pon_dir=argv$pondir, normalOrdinal=argv$normalordinal, tumourOrdinal=tumourordinal)
! }
! 
  # shadow breakpoint removed due to initial mapq20 filter reducing FP rate
  # bpfiltered = .addFilter(bpfiltered, "shadow", is_shadow_breakpoint(bpgr, begr, full_vcf))
  
