diff -crB pipeline-4.1-orig/lib/HMF/Pipeline/Functions/Validate.pm pipeline-4.1/lib/HMF/Pipeline/Functions/Validate.pm
*** a/lib/perl5/site_perl/5.26.1/HMF/Pipeline/Functions/Validate.pm	2018-08-13 11:03:15.073206062 +0200
--- b/lib/perl5/site_perl/5.26.1/HMF/Pipeline/Functions/Validate.pm	2018-08-13 11:03:31.383520981 +0200
***************
*** 164,174 ****
  
  sub verifyReadGroups {
      my ($read_groups, $bam_reads) = @_;
- 
-     my %header_rgids = map { $_->{tags}{ID} => 1 } @$read_groups;
-     my %reads_rgids = map { $_->{tags}{RG}{value} => 1 } @$bam_reads;
-     my @unknown_rgids = grep { not exists $header_rgids{$_} } keys %reads_rgids;
-     confess "read group IDs from read tags not in BAM header:\n\t" . join "\n\t", @unknown_rgids if @unknown_rgids;
      return 1;
  }
  
--- 164,169 ----
