diff -crB manta-1.1.0.release_src/src/c++/lib/manta/ReadGroupStatsUtil.cpp manta-1.1.0.release_src_patched/src/c++/lib/manta/ReadGroupStatsUtil.cpp
*** a/src/c++/lib/manta/ReadGroupStatsUtil.cpp	1970-01-01 01:00:00.000000000 +0100
--- b/src/c++/lib/manta/ReadGroupStatsUtil.cpp	2019-03-05 12:48:07.990754976 +0100
***************
*** 242,252 ****
              {
                  std::ostringstream oss;
                  oss << "ERROR: Too few observations (" << _totalOrientCount << ") to determine pair orientation for " << _rgLabel << "'\n";
!                 BOOST_THROW_EXCEPTION(LogicException(oss.str()));
              }
  
              const unsigned minMaxCount(static_cast<unsigned>(minMaxFrac*_totalOrientCount));
!             if (_orientCount[maxIndex] < minMaxCount)
              {
                  const unsigned maxPercent((_orientCount[maxIndex]*100)/_totalOrientCount);
                  std::ostringstream oss;
--- 242,252 ----
              {
                  std::ostringstream oss;
                  oss << "ERROR: Too few observations (" << _totalOrientCount << ") to determine pair orientation for " << _rgLabel << "'\n";
!                 //BOOST_THROW_EXCEPTION(LogicException(oss.str()));
              }
  
              const unsigned minMaxCount(static_cast<unsigned>(minMaxFrac*_totalOrientCount));
!             if (_orientCount[maxIndex] < minMaxCount && _totalOrientCount != 0)
              {
                  const unsigned maxPercent((_orientCount[maxIndex]*100)/_totalOrientCount);
                  std::ostringstream oss;
