diff -urB manta-1.1.0.release_src-orig/CMakeLists.txt manta-1.1.0.release_src/CMakeLists.txt
--- a/CMakeLists.txt	1970-01-01 01:00:01.000000000 +0100
+++ b/CMakeLists.txt	2020-09-28 21:39:06.818349245 +0200
@@ -195,7 +195,8 @@
 add_custom_target(${THIS_OPT} ALL)
 
 # redist includes
-include ("${THIS_MODULE_DIR}/boost.cmake")
+set(Boost_LIBRARIES "-lboost_chrono -lboost_date_time -lboost_filesystem -lboost_program_options -lboost_regex -lboost_serialization -lboost_system -lboost_timer -lboost_unit_test_framework")
+ADD_DEFINITIONS(-DBOOST_TEST_DYN_LINK)
 
 if (NOT WIN32)
     # global source QC -- don't allow non-ascii chars in source files:
