diff -crB ete2-2.3.10/setup.py ete2-2.3.10-patched/setup.py
*** a/setup.py	2015-11-05 20:18:33.000000000 +0100
--- b/setup.py	2018-03-26 15:35:21.660425035 +0200
***************
*** 168,181 ****
              print " Warning:\033[93m Optional library [%s] could not be found \033[0m" %mname
              print "  ",msg
              missing = True
-     
-     notwanted = set(["-h", "--help", "-n", "--dry-run"])
-     seen = set(_s.script_args)
-     wanted = set(["install", "bdist", "bdist_egg"])
-     if TRACKINSTALL and (wanted & seen) and not (notwanted & seen):
-         try:
-             welcome = urllib2.quote("New alien in earth!")
-             urllib2.urlopen("http://etetoolkit.org/static/et_phone_home.php?ID=%s&VERSION=%s&MSG=%s"
-                             %(ETEID, ETE_VERSION, welcome))
-         except urllib2.HTTPError, e: 
-             pass
--- 168,170 ----
